lib/perl/OVH/Bastion/allowkeeper.inc

```diff
@@ -556,6 +563,27 @@ sub access_modify {
             return R('ERR_CANNOT_OPEN_FILE', msg => "File '$file' cannot be written to");
         }
 
+        # if we're being asked to clear, it's pretty straightforward
+        if ($action eq 'clear') {
+            if (open(my $fh_file, '>', $file)) {
+                close($fh_file);
+                OVH::Bastion::syslogFormatted(
+                    severity => 'info',
+                    type     => 'acl',
+                    fields   => [
+                        ['action',  'clear'],
+                        ['type',    $params{'way'}],
+                        ['group',   $shortGroup],
+                        ['account', $params{'account'}],
+                    ]
+                );
+                return R('OK', msg => "Accesses successfully cleared");
+            }
+            else {
+                return R('ERR_CANNOT_OPEN_FILE', msg => "Unable to write open $file");
```

Comment:
```suggestion
                return R('ERR_CANNOT_OPEN_FILE', msg => "Unable to open $file");
```

do you want to log `$!`?

Reply:
Not returned to the user, but yep a `warn_syslog()` is a good idea!